classes/01class/exercises/c01-aws01/fabioacp/ANSWER.md

```diff
@@ -0,0 +1,54 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+```bash
+# Create a KeyPair
+aws ec2 create-key-pair --key-name MyKeyPair --query 'KeyMaterial' --output text > MyKeyPair.pem
+
+# Create a EC2 instance with a public IP
+aws ec2 run-instances \
+    --image-id ami-088ff0e3bde7b3fdf \
+    --instance-type t2.micro \
+    --count 1 \
+    --subnet-id subnet-a50917ec \
+    --key-name MyKeyPair \
+    --security-group-ids sg-0983070dc820d22e6 \
```

Comment:
where did this security group came from?

Reply:
I created it using the console. Should I use the CLI instead?